hatch/src/generators/tup/catch_definition.rs

```diff
@@ -0,0 +1,3 @@
+pub fn make_catch_definition_string() -> String {
+    String::from("#define CATCH_CONFIG_MAIN\n#include \"catch.hpp\"")
+}
```

Comment:
This is what I'm talking about. All we are doing typically is returning a string. This file is 3 lines. It used to be 17.